scripts/exrrfs_bufrsnd.sh

```diff
@@ -269,12 +277,13 @@ ln -sf ./itag                              fort.11
 
   echo DONE $fhr at `date`
 
-  mv $DATA/bufrpost/profilm.c1.${tmmark} $DATA/profilm.c1.${tmmark}.f${fhr}
+  cp $DATA/profilm.c1.${tmmark} $DATA/profilm.c1.${tmmark}.f${fhr}
```

Comment:
This PR not only changes "mv" to "cp" but also changes the directory structure (i.e. removes the "bufrpost" subdir).  why do rrfs and rtma have different treatments here?